caddytls/certificates.go

```diff
@@ -128,7 +142,8 @@ func cacheUnmanagedCertificatePEMFile(certFile, keyFile string) error {
 //
 // This function is safe for concurrent use.
 func cacheUnmanagedCertificatePEMBytes(certBytes, keyBytes []byte) error {
-	cert, err := makeCertificate(certBytes, keyBytes)
+	// XXX: Get a Config and pass the real CTLogURLs
```

Comment:
Probably the way to do this for now (and for the below `XXX:`) is to pass the Config into this function so its log URLs can be passed to makeCertificate. I'm still thinking if the actual submitting and stapling the SCT should be done elsewhere so we don't have to lob the Config everywhere but it won't stop a merge, we can refactor later.